firefox_ios/views/firefox_ios_dau.view.lkml

```diff
@@ -1,14 +1,25 @@
 include: "//looker-hub/combined_browser_metrics/views/active_users_aggregates.view.lkml"
```

Comment:
you don't need to include the view if you are not using it.
This comment applies to all views except the one on the feature_usage_metrics file, where you've extended the view.